cookbook/metadata.rb

```diff
@@ -0,0 +1,11 @@
+name             'baragon'
+maintainer       'EverTrue, Inc.'
+maintainer_email 'eric.herot@evertrue.com'
```

Comment:
if this was in its own repo, i'd say use EverTrue without hesitation. if you guys are willing to maintain this cookbook, it's probably fine to leave it EverTrue (i personally interpret "maintainer" as "maintainer of this cookbook"). if you don't care very much, let's make it HubSpot. (definitely keep yourself in maintainer_email)